guacamole/src/main/webapp/app/login/directives/login.js

```diff
@@ -102,6 +102,13 @@ angular.module('login').directive('guacLogin', [function guacLogin() {
          */
         $scope.submitted = false;
 
+        /**
+         * The field that needs to be focused
```

Comment:
JSDoc comments need to end with a period.

> The field that needs to be focused

This feels too tightly coupled to the actual internal implementation. It is ultimately used to focus a field, yes, but I don't think this level of things should be aware of that. The property is named nicely based on its high-level use: `relevantField`. Is there a better way to phrase this which captures that high-level intent?